Zend/zend_API.h

```diff
@@ -985,11 +980,7 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_wrong_callback_exception(int num, cha
 			    (UNEXPECTED(_num_args > _max_num_args) && \
 			     EXPECTED(_max_num_args >= 0))) { \
 				if (!(_flags & ZEND_PARSE_PARAMS_QUIET)) { \
-					if (_flags & ZEND_PARSE_PARAMS_THROW) { \
-						zend_wrong_parameters_count_exception(_min_num_args, _max_num_args); \
-					} else { \
-						zend_wrong_parameters_count_error(_min_num_args, _max_num_args); \
-					} \
+					zend_wrong_parameters_count_exception(_min_num_args, _max_num_args); \
```

Comment:
The RFC talks about throwing TypeError, so why did you only use the "count_*exception*" here, instead of repurposing "count_*error*" ?

Reply:
Agreed, I've switched these to use the `_error` terminology.